proposals/spiffe.adoc

```diff
@@ -0,0 +1,140 @@
+== SPIFFE
+
+*Name of project*: SPIFFE
+
+*Description*:
+
+With microservices, container orchestrators, and cloud computing leading to the deployment of increasingly dynamic and heterogeneous production environments, conventional network and application security practices struggle to scale under such distributed design patterns.
+
+Further, engineers must be involved in how applications are deployed and managed in such environments; and operations teams require deeper visibility into managed applications.
+
+As we move to a more evolved security stance, we must create technology frameworks that enable the aforementioned to play active roles in easily building secure, distributed applications. **SPIFFE (aka the “Secure Production Identity Framework for Everyone”)** is one such framework.
+
+SPIFFE comprises three (3) components:
```

Comment:
Note for the record: We believe we should contribute SPIRE together with SPIFFE, as one project.

One project is simpler, and emphasizes that SPIFFE is not just a spec, but also has a working implementation. TUF also has a reference implementation.

Istio will also have a SPIFFE implementation, which will help ensure that multiple implementations of the spec will remain possible.